### DISCUSSION TITLE
How to change the default web ui port?

### DISCUSSION THREAD

**Discussion by farfromuman** [zwave-js/zwave-js-ui#3837] [Q&A] [answered]
I am using the windows zwave-js-ui and would like to run a second instance, but I can't find where to change the default port.

**Comment by kpine** [comment, marked as answer]
Set the `PORT` environment variable. https://zwave-js.github.io/zwave-js-ui/#/guide/env-vars

**Reply by farfromuman** [reply]
Thank You.
